_data/boards.yml

```diff
@@ -762,97 +1036,142 @@
   boards:
   - name: ARMED
     brief: Arm'ed STM32F4-based controller
+    since: 2.0.0
   - name: RUMBA32_V1_0
     brief: RUMBA32 STM32F446VE based controller from Aus3D
+    since: 2.0.0
   - name: RUMBA32_V1_1
     brief: RUMBA32 STM32F446VE based controller from Aus3D
+    since: 2.0.4
   - name: RUMBA32_MKS
     brief: RUMBA32 STM32F446VE based controller from Makerbase
+    since: 2.0.4
   - name: RUMBA32_BTT
     brief: RUMBA32 STM32F446VE based controller from BIGTREETECH
+    since: 2.0.9.2
   - name: BLACK_STM32F407VE
     brief: BLACK_STM32F407VE
+    since: 2.0.0
   - name: BLACK_STM32F407ZE
     brief: BLACK_STM32F407ZE
+    since: 2.0.0
   - name: BTT_SKR_MINI_E3_V3_0_1
     brief: BigTreeTech SKR Mini E3 V3.0.1 (STM32F401RC)
+    since: 2.1.2
   - name: BTT_SKR_PRO_V1_1
     brief: BigTreeTech SKR Pro v1.1 (STM32F407ZG)
+    since: 2.0.4
   - name: BTT_SKR_PRO_V1_2
     brief: BigTreeTech SKR Pro v1.2 (STM32F407ZG)
+    since: 2.0.6
   - name: BTT_BTT002_V1_0
     brief: BigTreeTech BTT002 v1.0 (STM32F407VG)
+    since: 2.0.4
   - name: BTT_E3_RRF
     brief: BigTreeTech E3 RRF (STM32F407VG)
+    since: 2.0.8
   - name: BTT_SKR_V2_0_REV_A
     brief: BigTreeTech SKR v2.0 Rev A (STM32F407VG)
+    since: 2.0.8.1
   - name: BTT_SKR_V2_0_REV_B
     brief: BigTreeTech SKR v2.0 Rev B (STM32F407VG/STM32F429VG)
+    since: 2.0.8.1
   - name: BTT_GTR_V1_0
     brief: BigTreeTech GTR v1.0 (STM32F407IGT)
+    since: 2.0.2
   - name: BTT_OCTOPUS_V1_0
     brief: BigTreeTech Octopus v1.0 (STM32F446ZE)
+    since: 2.0.8.1
   - name: BTT_OCTOPUS_V1_1
     brief: BigTreeTech Octopus v1.1 (STM32F446ZE)
+    since: 2.0.9
   - name: BTT_OCTOPUS_PRO_V1_0
     brief: BigTreeTech Octopus Pro v1.0 (STM32F446ZE / STM32F429ZG)
+    since: 2.0.9.3
   - name: LERDGE_K
     brief: Lerdge K (STM32F407ZG)
+    since: 2.0.0
   - name: LERDGE_S
     brief: Lerdge S (STM32F407VE)
+    since: 2.0.6
   - name: LERDGE_X
     brief: Lerdge X (STM32F407VE)
+    since: 2.0.0
   - name: FYSETC_S6
     brief: FYSETC S6 (STM32F446VE)
+    since: 2.0.0
   - name: FYSETC_S6_V2_0
     brief: FYSETC S6 v2.0 (STM32F446VE)
+    since: 2.0.6.1
   - name: FYSETC_SPIDER
     brief: FYSETC Spider (STM32F446VE)
+    since: 2.0.8
   - name: FLYF407ZG
     brief: FLYmaker FLYF407ZG (STM32F407ZG)
+    since: 2.0.1
   - name: MKS_ROBIN2
     brief: MKS Robin2 V1.0 (STM32F407ZE)
+    since: 2.0.1
   - name: MKS_ROBIN_PRO_V2
     brief: MKS Robin Pro V2 (STM32F407VE)
+    since: 2.0.8
   - name: MKS_ROBIN_NANO_V3
     brief: MKS Robin Nano V3 (STM32F407VG)
+    since: 2.0.8
   - name: MKS_ROBIN_NANO_V3_1
     brief: MKS Robin Nano V3.1 (STM32F407VE)
+    since: 2.0.9.4
   - name: MKS_MONSTER8_V1
     brief: MKS Monster8 V1 (STM32F407VE)
+    since: 2.0.9.5/2.1.1
   - name: MKS_MONSTER8_V2
     brief: MKS Monster8 V2 (STM32F407VE)
+    since: 2.0.9.5/2.1.1
```

Comment:
These "since" versions are a bit different since we can't simply say "since `2.0.9.5`" due to these names not existing in the `2.1.x` branch until `2.1.1`.